recipes-bsp/u-boot/files/ae350-ax45mp/0002-spl-Align-device-tree-blob-address-at-8-byte-boundar.patch

```diff
@@ -0,0 +1,38 @@
+From 933ad8a59f7fd9b2088badc3e97167d750a40b5a Mon Sep 17 00:00:00 2001
+From: Bin Meng <bmeng.cn@gmail.com>
+Date: Mon, 12 Jul 2021 11:52:31 +0800
+Subject: [PATCH] spl: Align device tree blob address at 8-byte boundary
+
+Since libfdt v1.6.1, a new requirement on the device tree address via:
+
+  commit 5e735860c478 ("libfdt: Check for 8-byte address alignment in fdt_ro_probe_()")
+
+must be met that the device tree must be loaded in to memory at an
+8-byte aligned address.
+
+Upstream-Status: Pending
+Signed-off-by: Bin Meng <bmeng.cn@gmail.com>
```

Comment:
what are plans for upstreaming it into u-boot ?

Reply:
Well.. there seems to be no solution on this [thread](https://patchwork.ozlabs.org/project/uboot/patch/20210712035231.26475-1-bmeng.cn@gmail.com/), but we did run into the misaligned issue without this patch.